yarn.lock

```diff
@@ -0,0 +1,165 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Hm, would prefer to not commit `yarn.lock`. Might make merging repositories less clean.